mzLib/UsefulProteomicsDatabases/ProteinXmlEntry.cs

```diff
@@ -225,7 +225,31 @@ public void ParseFeatureEndElement(XmlReader xml, IEnumerable<string> modTypesTo
             }
             else if (FeatureType == "peptide" || FeatureType == "propeptide" || FeatureType == "chain" || FeatureType == "signal peptide")
             {
-                ProteolysisProducts.Add(new ProteolysisProduct(OneBasedBeginPosition, OneBasedEndPosition, FeatureType));
+                string type = FeatureType;
```

Comment:
Could you add a comment before this block to describe what's going on?

Reply:
Mostly noting that this block produces descriptions relevant to top-down biomarkers following the `(` character